package-lock.json

```diff
@@ -9,10 +9,10 @@
             "dev": true,
```

Comment:
package lock updates aren't needed to change the CSS

Reply:
yes, this hapenned when npm installing less

Reply:
with git you can:
```
git add public/css/index.css
git add  public/less/_repository.less
git commit
```
that will only add specific files to your commit.